### DISCUSSION TITLE
Where is the auto-smart mode algorithme?

### DISCUSSION THREAD

**Discussion by SomeMinecraftModder** [ThioJoe/YT-Spammer-Purge#384] [Q&A] [answered]
Does somebody know what file and at what line is the algorithme of the auto-smart mode? Intersted in how it work.

**Comment by TechStudent10** [comment, marked as answer]
I think it's
https://github.com/ThioJoe/YT-Spammer-Purge/blob/main/YouTubeSpammerPurge.py#L504

**Reply by SomeMinecraftModder** [reply]
Thanks you!

**Reply by TechStudent10** [reply]
@SomeMinecraftModder No problem. Just looked some more another reference: https://github.com/ThioJoe/YT-Spammer-Purge/blob/main/YouTubeSpammerPurge.py#L593
